### PR TITLE
2693 - Radio and 2715, 2725, 2709, 2691

### PR DESCRIPTION
**Pull request by Fwerfad**
Documentation and images for:
2691 Checkbox
2693 - Radio
2725 - Click Away Listener
2715 - Avatar
2709 - Card

### CODE REVIEW

File: source/includes/_documentation.md [hunk 1]
```diff
@@ -23864,47 +23864,449 @@ Note: domain is read from test.properties automatically. <br><br><br><br><br>
...
+        for (int i = 1; i < 3; i++) {
+            checkboxTestLogic(
+                    checkboxPage.formControlLabelCheckbox.get(i),
+                    i != 2 && i != 7 ? "MuiCheckbox-colorSecondary" :
+                            i == 2 ? "MuiCheckbox-colorPrimary" : "jss3");
+            textTestLogic(checkboxPage.formControlLabelText.get(i), expectedTexts[i - 1]);
+        }
+    }
+
+    @Test
```
Comment (LunAndrey) [inline source/includes/_documentation.md:23932]: delete this test

File: source/includes/_documentation.md [hunk 2]
```diff
@@ -23864,47 +23864,449 @@ Note: domain is read from test.properties automatically. <br><br><br><br><br>
...
+                    checkboxPage.formGroupCheckbox.get(secondCheckboxIndex));
+            textTestLogic(checkboxPage.formGroupText.get(i), expectedTexts[i - 1]);
+        }
+        groupTestErrorLogic(
+                checkboxPage.formGroupCheckbox.get(1),
+                checkboxPage.formGroupCheckbox.get(2),
+                checkboxPage.errorMessage.get(2));
+    }
+
+    @Test
```
Comment (LunAndrey) [inline source/includes/_documentation.md:23949]: delete this test

File: source/includes/_documentation.md [hunk 3]
```diff
@@ -23864,47 +23864,449 @@ Note: domain is read from test.properties automatically. <br><br><br><br><br>
...
+                lastRadioText.has().text(containsString(currentRadioButton.text()));
+            }
+            else
+                timer.wait(() -> currentRadioButton.has().classValue(containsString("Mui-disabled")));
+            currentRadioButtonLabel.has().text(labels.get(i - 1));
+        }
+    }
+
+    @Test
+    public void labelPlacementTest() {
```
Comment (LunAndrey) [inline source/includes/_documentation.md:24279]: delete this test

File: source/includes/_documentation.md [hunk 4]
```diff
@@ -23864,47 +23864,449 @@ Note: domain is read from test.properties automatically. <br><br><br><br><br>
...
+            Button currentRadioButton = labelPlacementButtons.get(i);
+            Button currentButtonClass = labelPlacementButtonsClass.get(i);
+            if (i != 4)
+                currentRadioButton.has().classValue(containsString(classes.get(i - 1)));
+            currentRadioButton.click();
+            timer.wait(() -> currentButtonClass.has().classValue(containsString("Mui-checked")));
+        }
+    }
+
+    @Test
```
Comment (LunAndrey) [inline source/includes/_documentation.md:24290]: delete this test

File: source/includes/_documentation.md (hunk 1 above)
Comment (Fwerfad) [reply source/includes/_documentation.md:23932]: done

File: source/includes/_documentation.md (hunk 2 above)
Comment (Fwerfad) [reply source/includes/_documentation.md:23949]: done

File: source/includes/_documentation.md (hunk 3 above)
Comment (Fwerfad) [reply source/includes/_documentation.md]: done

File: source/includes/_documentation.md (hunk 4 above)
Comment (Fwerfad) [reply source/includes/_documentation.md]: done

**Pull request merged**